docs/api/datasets.rst

```diff
@@ -227,6 +227,11 @@ Potsdam
 
 .. autoclass:: Potsdam2D
 
+ReforesTree
+^^^^^^^^^^^
+
+.. autoclass:: ReforesTree
```

Comment:
Can you also add a line to the `docs/api/non_geo_datasets.csv` file describing this dataset?